src/test/java/com/qa/restassured/VerifyNameWithStaticImports.java

```diff
@@ -0,0 +1,19 @@
+package com.qa.restassured;
+
+import static io.restassured.RestAssured.*;
+
+import static org.hamcrest.Matchers.*;
+
+public class VerifyNameWithStaticImports {
+
```

Comment:
I think the @Test annotation is missing here too

Reply:
Modify